ziti/cmd/verify/common.go

```diff
@@ -0,0 +1,48 @@
+/*
+	Copyright NetFoundry Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+package verify
+
+import (
+	log "github.com/sirupsen/logrus"
+	"runtime"
+)
+
+type controller struct {
+	user string
+	pass string
+	host string
+	port string
+}
+
+func configureLogFormat() {
+	log.SetFormatter(&log.TextFormatter{
```

Comment:
I haven't seeen something call `log.SetFormatter` in a long time. Most of our code calls `pfxlog.Logger()` and gets the globally configured logger.

I don't understand why we would need to do it here over use the established patterns, unless there is something I'm missing.

Reply:
largely it was just an aesthetics thing...  removing the timestamp, colorizing the level and removing the package/function. 

example before/after setting the format. if there's a way to remove the timestamp/package/function though, it'd work. I just didn't find it in options. i admit, i might not have known what to look for

![image](https://github.com/user-attachments/assets/cf6c0e56-eb24-48cd-abc0-3cadf05f5144)

It's also somewhat confusing to have a clone of it in the openziti repo but not be using it so i just kept it to "one fewer import"